tests/5.1/target/test_target_memcpy_async_no_obj.F90

```diff
@@ -0,0 +1,104 @@
+!===--- test_target_memcpy_async_no_obj.F90 -------------------------------===//
+!
+! OpenMP API Version 5.1 Nov 2020
+!
+!  Inspired from OpenMP 5.1 Examples Doc, 5.16.4 & 8.9
+!  This test utilizes the omp_target_memcpy_async construct to
+!  allocate memory on the device asynchronously. The construct
+!  uses 0 for 'depobj_count', so that the clause is not dependent
+!  and memory is therefore copied synchronously.
+!
+!//===---------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_memcpy_async_no_obj
+  USE iso_fortran_env
+  USE, INTRINSIC :: iso_c_binding
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(test_memcpy_async_no_obj() .NE. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_memcpy_async_no_obj()
+    INTEGER :: errors, i
+    DOUBLE PRECISION, TARGET, ALLOCATABLE :: arr(:)
+    DOUBLE PRECISION, POINTER :: fptr(:)
```

Comment:
If the suggestion below is applied, this becomes unused.
```suggestion
```

Reply:
@tob2 
I would like to run the tests with a up to date GCC and GFORTRAN with Nvidia and AMD GPUs, but I don't have access to any system that have them. Do you have a script to build it that you can share with me?